packaging_apps_manifest.md

```diff
@@ -62,6 +63,8 @@ The `manifest.json` file defines the app's constants, a bunch of values that Yun
 
 * **url**: software website.
 
+* **version**: version of upstream software which is installed. Must be a string.
```

Comment:
Personnaly i don't put the axact upstream version number. Because I used it to manage the upgrade procedure of the package. So, in my case it is the package version number, I create it like this:
UPSTREAM_VERSION_NUMBER-PACKAGE_NUMBER